sound/soc/sdw_utils/soc_sdw_utils.h

```diff
@@ -51,6 +51,16 @@ struct sof_sdw_codec_info {
 			    struct snd_soc_codec_conf **codec_conf);
 };
 
+struct mc_private {
+	struct snd_soc_card card;
+	struct snd_soc_jack sdw_headset;
+	struct device *headset_codec_dev; /* only one headset per card */
+	struct device *amp_dev1, *amp_dev2;
+	void *intel_mc_private;
```

Comment:
just use 'private', Intel and AMD will use the same void pointer differently and cast accordingly.

Reply:
will fix it